wheatley/parsing.py

```diff
@@ -13,21 +18,21 @@ class PealSpeedParseError(ValueError):
     cannot be parsed by the parser.
     """
 
-    def __init__(self, peal_speed_string, message):
+    def __init__(self, peal_speed_string: str, message: str):
         super().__init__()
 
         self.peal_speed_string = peal_speed_string
         self.message = message
 
-    def __str__(self):
+    def __str__(self) -> str:
         return f"Error parsing peal speed '{self.peal_speed_string}': {self.message}"
 
 
-def parse_peal_speed(peal_speed: str):
+def parse_peal_speed(peal_speed: str) -> int:
     """
     Parses a peal speed written in the format /2h58(m?)/ or /XXX(m?)/ into a number of minutes.
     """
-    def exit_with_message(error_text):
+    def exit_with_message(error_text: str) -> None:
```

Comment:
```suggestion
    def exit_with_message(error_text: str) -> NoReturn:
```

Reply:
Aha this is a thing?! Ok I'll do a cheeky find/replace...